scripts/helmcharts/openreplay/appset-values.yaml

```diff
@@ -0,0 +1,5 @@
+repoUrl: https://github.com/openreplay/openreplay
```

Comment:
Please remove this file, as its not needed

Reply:
fixed